aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/RunFaceLivenessSession.kt

```diff
@@ -128,7 +128,7 @@ internal class RunFaceLivenessSession(
 
     private fun getChallengeId(): String = livenessWebSocket.challengeId
 
-    private fun getChallengeType(): FaceLivenessChallengeType = livenessWebSocket.challengeType
+    private fun getChallengeType(): FaceLivenessChallengeType = livenessWebSocket.challengeType!!
```

Comment:
Lets remove this method and instead pass challenge type through to:

```
onSessionInformationReceived.accept(
    response.serverSessionInformationEvent.sessionInformation
)
```
We can change type of the consumer to contain a class wrapping SessionInformation and ChallengeType `val onSessionInformationReceived: Consumer<SessionInformation>`